src/sm/networkmanager/exec.hpp

```diff
@@ -11,24 +11,9 @@
 
 #include <core/common/tools/error.hpp>
 
+#include "itf/exec.hpp"
```

Comment:
In itf folder we should put interfaces that are used by other modules. If you have some interfaces that internal for module and designed for unit testing purpose, I guess they should be in the module folder.